etc/kuiper.yaml

```diff
@@ -1,6 +1,4 @@
 basic:
-  # debug | info | warn | error | fatal | panic
-  logLevel: info
   # true|false, with debug level, it prints more debug info
```

Comment:
Do not change this file. We've added new options in this release

Reply:
@ngjaying File recovered